stellargraph/core/graph.py

```diff
@@ -39,6 +39,70 @@
 NeighbourWithWeight = namedtuple("NeighbourWithWeight", ["node", "weight"])
 
 
+def _features(element_data, unique, name, ids, type, use_ilocs):
```

Comment:
If this is going to move to the top level to be used by other files, maybe we could make the name a little more descriptive, to provide more context, e.g.

```suggestion
def extract_element_features(element_data, unique, name, ids, type, use_ilocs):
```

By _not_ including it in the `__all__` or the docs, it's still effectively private to the Stellargraph module.